### PR TITLE
Fixes #893 Add visit counter for Umple Online

### LINKED ISSUE #893: Add visit counter to UmpleOnline
Add a visit counter to Umple online using the PhP code. 

The wording should be after 'Report an issue' and should say. "n visits since MMM 2016", where n is updated, and mmm is the month it was added.

I suggest using visit counter php code at http://justintadlock.com/web-design/counter
(Give him credit in a comment)

Why do this: It might build some excitement

### PR DESCRIPTION
**Pull request by glorialaw**
Fixes #893. Visit counter is added on the bottom right hand corner.

### CODE REVIEW

File: umpleonline/umple.php [hunk 1]
```diff
@@ -573,4 +572,7 @@
       ); //
   </script>
 </body>
+	<div class="visitors-count" align="right">
```
Comment (TimLethbridge) [inline umpleonline/umple.php]: Although this seems to work, any div should be inside the body tag, so the end body tag just above needs to be below the end of this div

**Review by TimLethbridge — COMMENTED** [review]
Note that I edited the pull request comment! You had left the template comment (I had to do this for your last pull request too).

**Comment by glorialaw** [comment]
I have fixed the umple.php and PR comment. Thanks for letting me know @TimLethbridge

**Review by TimLethbridge — CHANGES_REQUESTED** [review]
I just realized one more thing: countlog.txt can't be in version control. Why not? It will get reset every time umpleonline is updated. Simply delete the file; in your checks for its readability, simply create it with a value of 1 if it does not exist. Also put countlog.txt in .gitignore so its creation and changes don't trigger git to suggest it as a changed file.

**Comment by glorialaw** [comment]
The changes have been made

**Comment by TimLethbridge** [comment]
This latest change doesn't seem to work. When trying it in docker, the counter doesn't update (but error messages are turned off)

So I tried this in my own local UmpleOnline, which should be the same as on the production server. My first result was: 
```
Warning: fopen(/Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/countlog.txt): failed to open stream: Permission denied in /Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/counter.php on line 13

Warning: fwrite() expects parameter 1 to be resource, boolean given in /Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/counter.php on line 14

Warning: fclose() expects parameter 1 to be resource, boolean given in /Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/counter.php on line 15

Warning: chmod(): No such file or directory in /Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/counter.php on line 18

Warning: fopen(/Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/countlog.txt): failed to open stream: No such file or directory in /Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/counter.php on line 20

Warning: fgets() expects parameter 1 to be resource, boolean given in /Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/counter.php on line 21

Warning: fclose() expects parameter 1 to be resource, boolean given in /Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/counter.php on line 22
1 visits since October 2018
Warning: fopen(/Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/countlog.txt): failed to open stream: Permission denied in /Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/counter.php on line 28

Warning: fwrite() expects parameter 1 to be resource, boolean given in /Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/counter.php on line 29

Warning: fclose() expects parameter 1 to be resource, boolean given in /Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/counter.php on line 30
```

Then I created the file manually and I got:
```
Warning: chmod(): Operation not permitted in /Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/counter.php on line 18
1 visits since October 2018
Warning: fopen(/Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/countlog.txt): failed to open stream: Permission denied in /Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/counter.php on line 28

Warning: fwrite() expects parameter 1 to be resource, boolean given in /Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/counter.php on line 29

Warning: fclose() expects parameter 1 to be resource, boolean given in /Users/tcl/Documents/7--Research/2-CRuiSE/codebase/googleumple/umple/umpleonline/counter.php on line 30
```

So the user has to manually create the file. I think the only way forward is to simply check for file existence and do nothing if the file is absent. Please make a change to that effect.

By the way, I pushed a change to .gitignore so when the .txt file is changed it does not get tracked as a change.

**Comment by glorialaw** [comment]
Thanks for letting me know! I have made the made the changes.

I tested the file adding part by deleting the countlog.txt file on my computer and those errors did not occur. However, when I first used the code from the website, I got those errors due to the incorrect file permissions on my system for the countlog.txt file.

**Review by TimLethbridge — APPROVED** [review]

**Pull request merged**